.env.example

```diff
@@ -1,6 +1,6 @@
 # For e2e tests:
 ENGINE_CONNECTION_STRING= # Format: https://<cluster_name>.<region>.kusto.windows.net
-DM_CONNECTION_STRING= # Optional - if not set, will infer from ENGINE_CONNECTION_STRING. Format: https://ingest-<cluster_name>.<region>.kusto.windows.net
+DM_CONNECTION_STRING= # Format: https://ingest-<cluster_name>.<region>.kusto.windows.net
```

Comment:
What issue did you encounter with this?

The only place we use it in the code infers it from the engine:
```python
        cls.dm_cs = get_env("DM_CONNECTION_STRING", default=cls.engine_cs.replace("//", "//ingest-"))
```